aptos-move/block-executor/src/executor.rs

```diff
@@ -290,6 +291,34 @@ where
         txn_fee_statements: &mut Vec<FeeStatement>,
     ) {
         while let Some(txn_idx) = scheduler.try_commit() {
+            let read_set = last_input_output
+                .read_set(txn_idx)
+                .expect("Read set must be recorded");
+            let mut execution_still_valid =
+                read_set.validate_delayed_field_reads(versioned_cache.aggregators(), txn_idx);
+            if execution_still_valid {
+                if let Some(aggregator_ids) = last_input_output.aggregator_v2_keys(txn_idx) {
+                    if let Err(e) = versioned_cache
+                        .aggregators()
+                        .try_commit(txn_idx, aggregator_ids.collect())
+                    {
+                        match e {
+                            CommitError::ReExecutionNeeded(_) => {
```

Comment:
we do have some committed values (i.e. converted to Value) in MVHashMap right now, but should be Ok.